### PR TITLE
[data value] audits

### PR DESCRIPTION
**Pull request by tokland**
Adds paginated endpoint `audit/dataValues`. We dismissed having conditional types to avoid complexity for paging yes/no, so we have to decide if we duplicate the endpoints or not.

### CODE REVIEW

File: src/api/audit.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+// Apparently, the official docs are wrong (they reference a skipPaging property).
+// Standard properties paging/page/pageSize work.
+export type DataValuesAuditOptions = {
+    ds?: Id[];
+    de?: Id[];
+    pe?: string[];
+    ou?: Id[];
+    auditType?: AuditType;
+    // paging: boolean;
```
Comment (SferaDev) [inline src/api/audit.ts:30]: @tokland Is it normal that this property is commented?

Comment (SferaDev) [reply src/api/audit.ts:30]:
```suggestion
```

Just tested, you force paging in line 15

**Review by SferaDev — APPROVED** [review]
Code LGTM + Tested. Results are consistent with API call

File: src/api/audit.ts (hunk 1 above)
Comment (tokland) [reply src/api/audit.ts]: > @tokland Is it normal that this property is commented?

Typical "just know that this property exist", but can be removed

**Pull request merged**